### ISSUE TITLE
[New build - EXPERIMENTAL] 20w50a: Planets Variety

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#3354]
This build introduces more planets type.
There is now Ice, lava, gas, sand, thunderstorm, terrestrial, moon-like and oceanic planets, each in several forms.
These are simple placeholder for now while waiting for a full fledged procedural planetary tech.

Bugfixes:
Drops not being cleared on quit to title.
#3345 #3093 Massive amount of drops causing FPS drop.
Raycast length issue on Space Habitat terrain.
#3348 Rare instance of camera freeze.

**Comment by tsunamayo** [comment]
Build is up! Cheers guys!

**Comment by chacalik** [comment]
![image](https://user-images.githubusercontent.com/4794243/101558894-ae7d5f00-399e-11eb-96cd-48c32d74cfa6.png)


Hey Tsuna, the build version ingame is wrong I guess

**Comment by BrickMacklin** [comment]
Tsuna if I am not mistaken the planets appear larger (which is great!) Can the habitat be made larger?

**Comment by ultrak2k** [comment]
I'd also like to see the hab grow larger!

**Comment by ProPeach** [comment]
These planets are genuinely beautiful! The Earth-like terrestrial types are my favourite

**Comment by Kaiser-Indrasil** [comment]
The thunderstorm planets are my favourite but they are all beautiful! Who made the textures for them, you or Grey Shuko?

**Comment by TIKIRobo** [comment]
It also could be interesting the rings rotated as now that we have "dynamic" planets it seems strange that the rings on them don't rotate. Also the same  could be said for habitats as they seems very still compared to these new planets.

**Comment by tsunamayo** [comment]
@chacalik thanks, I have just repushed the build with the correct build id.

**Comment by tsunamayo** [comment]
Yes I will add rotation. @Kaiser-Indrasil this is all placeholder for now, no I didnt spent much time / budget on them as the end goal is to have procedural planet you can land on, so they are just an old asset I bought a long time ago.

**Comment by Kaiser-Indrasil** [comment]
I see, it's smart. Can we expect that your plans for the final iteration of the planets system will be based on these ones?
